README.md

```diff
@@ -1,31 +1,62 @@
 
-Development of the next version of the OpenAPI Specification is guided by the [OAI Technical Contributors Board](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the OpenAPI Specification, this branch will be merged to master.
+Use cases for machine-readable API interfaces include interactive documentation, code generation for documentation, client, and server, as well as automated test cases. OpenAPI descriptions describe APIs via YAML or JSON documents that adhere to the OpenAPI Specification documented in this repository.  These documents can either be produced and served statically, or be generated dynamically from your application.
 
-The current process for development of the OpenAPI Specification is described in [Development Guidelines](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/DEVELOPMENT.md).
+Without going into a long history of interfaces to Web Services, this is not the first attempt to do so.  We can learn from CORBA, WSDL and WADL.  These specifications had good intentions but were limited by proprietary vendor-specific implementations, being bound to a specific programming language, and goals which were too open-ended.  In the end, they failed to gain traction.
+
+The OpenAPI Specification does not require you to rewrite your existing API.  It does not require binding any software to a service--the service being described may not even be yours.  It does, however, require the capabilities of the service be described in the structure of the OpenAPI Specification.  Not all services can be described by OpenAPI--this specification is not intended to cover every possible use-case of a REST API. The OpenAPI Specification does not define a specific development process such as design-first or code-first.  It does facilitate either technique by establishing clear interactions with a REST API.
+
+This GitHub project is the starting point for OpenAPI.
+Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like,
+and some general information regarding the project.
+
+## Current Version - 3.0
+
+The current version of the OpenAPI specification is 3.0 - and you can find it [here](versions/3.0.md).
```

Comment:
Suggest: `The current version of the OpenAPI specification is [OpenAPI Specification 3.0](versions/3.0.md).`